h2o-r/h2o-DESCRIPTION.template

```diff
@@ -55,6 +55,10 @@ Suggests: ggplot2,
           slam,
           bit64 (>= 0.9.7),
           data.table (>= 1.9.8),
+          open3d,
```

Comment:
```suggestion
```

Reply:
This is a function name, not a library name. The library is `persp3D`.

Reply:
Erin:  

I removed all the packages and only keep rgl, plot3Drgl.